crates/orderbook/openapi.yml

```diff
@@ -51,6 +51,32 @@ paths:
           application/json:
             schema:
               $ref: "#/components/schemas/OrderCreation"
+    delete:
+      summary: Cancels multiple orders by marking it invalid with a timestamp.
```

Comment:
@anxolin, this is the current proposal for cancelling multiple orders with a single signature and API call.

It would be something like:
```js
const orderUids = [...];
await fetch(`${base}/api/v1/orders`, {
  method: "DELETE",
  body: {
    orderUids,
    signingScheme: "eip712",
    signature: await signer._signTypedData(
      domainSeparator,
      {
        "OrderCancellations": [
          { name: "orderUids", type: "bytes[]" },
        ],
      },
      { orderUids },
    ),
  },
});
```